config/test.js

```diff
@@ -32,7 +32,7 @@ module.exports = {
   },
 
   cache: {
-    volumeCachePath: path.join(__dirname, '..', 'test', 'tmp')
+    volumeCachePath: path.join('/tmp', 'test-cache')
```

Comment:
is the reason for polluting things outside of the docker-worker directory so that we have a determinate path for tests?